pmm-app/tests/QAN/steps/qanActions.js

```diff
@@ -425,4 +426,17 @@ module.exports = {
 
     return resultsCount[2];
   },
+
+  async verifySelectedFilters(filters) {
+    I.click(qanPage.fields.showSelected);
+    I.waitForVisible(qanPage.fields.filterName, 20);
+    const getFilter = await I.grabTextFrom(qanPage.fields.filterName);
+    for (let i = 0; i <= filters.length - 1; i++) {
+      if (getFilter[i].includes(filters[i])) {
+        //do nothing
+      } else {
+        assert.fail(`The filter: ${filters[i]} have not be found!`);
+      }
```

Comment:
```suggestion
      if (!getFilter[i].includes(filters[i])) {
        assert.fail(`The filter '${filters[i]}' has not been found!`);
      }
```
1/ shorter
2/ spelling
3/ surround the filter with ''